en/test_and_ci/README.md

```diff
@@ -2,22 +2,10 @@
 
 PX4 offers extensive unit testing and continuous integration facilities. This page provides an overview.
 
-## Testing on the local machine
+* [Unit Tests](../test_and_ci/unit_tests.md)
+* [Continuous Integration (CI)](../test_and_ci/continous_integration.md)
+* [Jenkins CI](../test_and_ci/jenkins_ci.md)
+* [Integration Testing](../test_and_ci/integration_testing.md)
+* [Docker](../test_and_ci/docker.md)
+* [Maintenance](../test_and_ci/maintenance.md)
```

Comment:
There is  a [VTOL testing](https://dev.px4.io/en/airframes_vtol/testing.html) doc in the sidebar here but not in this list. I think it should be deleted - it is very out of date - I am still trying to get it updated here: https://github.com/PX4/px4_user_guide/pull/74

FMI More generally though, are there/should there be topics about testing that do not fix under this whole "system testing/CI" banner? To put it in context, most testing and tuning stuff should be in the user guide, because that is for people working on existing airframes and hardware. I wonder though if you are creating a *new* airframe or integrating a new sensor whether there is specific testing we need to document.

Reply:
it really depends. if there is a new airframe, then I think that kind of testing is something different. if someone integrates a new sensor, then he should consider unit testing. 
I think everything that is related to software changes require testing within this section. Hardware testing and flight performance testing is a different subject and needs to live somewhere else.

Reply:
So perhaps "Platform Testing & CI" to restrict scope to just the automated testing?